test/Microsoft.TestPlatform.TestUtilities/Microsoft.TestPlatform.TestUtilities.csproj

```diff
@@ -21,6 +27,10 @@
   <ItemGroup>
     <PackageReference Include="Moq" Version="$(MoqVersion)" />
     <PackageReference Include="MSTest.TestFramework" Version="$(MSTestFrameworkVersion)" />
+    <PackageReference Include="Microsoft.CodeAnalysis.BannedApiAnalyzers" Version="3.3.3">
```

Comment:
Shall we put the version inside the `eng\Versions.props` or `scripts\build\TestPlatform.Dependencies.props`?

Reply:
I copied this from your code I think 😁

Reply:
Ooopsie :)